wrappers/MATLAB/refpropm.m

```diff
@@ -691,4 +706,32 @@
     end
 end
 
+
+function proto = get_header_path()
+    % Systematic lookup method to find possible header file
+    % Start with the BasePath, then look through unix standard places
+    % and end by looking in MATLAB path.
+    if exist('rp_proto64', 'file')

```

Comment:
Proto files are required for platforms without a valid compiler or when using the MATLAB Runtime Compiler to execute compiled MATLAB code (using a header file is not allowed). For this reason, the function starts by looking for a proto in the path. Otherwise, it looks for the header in a few possible locations.
I will edit the MATLAB README file to explain how to generate the proto function (if required).